dbt_subprojects/dex/models/trades/base/platforms/xchange_base_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'xchange_base',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v2_trades(
+        blockchain = 'base',
+        project = 'xchange',
+        version = '1',
+        Pair_evt_Swap = source('xchange_base', 'XchangePair_evt_Swap'),
```

Comment:
There are 548 records in `XchangePair_evt_Swap` currently:
https://dune.com/queries/3994474

However, only 2 listed in the base trades:
https://dune.com/queries/3994479

Reply:
Is it possible that those 2 in the test schema table are just the ones from my seed file? I get a lot more results when I join the Swap and PairCreated table, which should be what we are doing to get most of the data in the Xchange base trades table. If that's not the case, I can add more transactions in the seed file if that helps troubleshoot. 

select count(*)
from xchange_base.XchangePair_evt_Swap t
inner join xchange_base.XchangeFactory_evt_PairCreated p
on t.contract_address=p.pair

Reply:
Actually, if that test schema table is just at the point in time when I created the pull request, and is not updated in real time, I think I know why there's only 2 transactions. I had just submitted the base XchangeFactory contract that day and it had just gotten approved, so it only had 1 PairCreated event at the time. Since it's an inner join, we only got the swaps for that particular pair, which was a pair (and swaps) that I had just created particularly for this test and only a short period of time before the dbt test ran. I believe there was a bit of a delay in populating the rest of the pairs in that table, but now it seems to be up to date. If we rerun the dbt test, I think the test schema table will have the same number of rows as the Swap table.

Reply:
Sorry for all the comments but I just saw you reran it earlier today and it generated a new test schema table and results look good! 

select * from
test_schema.git_dunesql_018c365_xchange_base_base_trades
limit 1000

Reply:
Yes, it works now.